crates/aiken-lang/src/format.rs

```diff
@@ -633,6 +642,43 @@ impl<'comments> Formatter<'comments> {
             .append("}")
     }
 
+    #[allow(clippy::too_many_arguments)]
+    fn definition_benchmark<'a>(
```

Comment:
I believe this could be unified with the formatting of fuzzer definitions as well.